packages/kbn-unified-field-list/src/components/field_item_button/field_item_button.scss

```diff
@@ -42,7 +42,7 @@
 
 .unifiedFieldListItemButton {
   width: 100%;
-
+  background: $euiColorEmptyShade;
```

Comment:
Page background is not always like on Discover page. For example in Discover mobile view or on the examples page `/app/unifiedFieldListExamples` it's grey and items background becomes visible:

<img width="400" alt="Screenshot 2024-03-12 at 13 23 25" src="https://github.com/elastic/kibana/assets/1415710/321fccd7-47d4-42d8-84e9-695fbd786dd9">

I'm not against it if it's a necessary change. @MichaelMarcialis wdyt?

Reply:
Now it seems to be broken in Lens
<img width="319" alt="Screenshot 2024-03-12 at 14 44 13" src="https://github.com/elastic/kibana/assets/1415710/effbebd5-db18-4de2-a303-33d3afb06734">

Reply:
Should be fine now! Example page:
<img width="379" alt="Screenshot 2024-03-14 at 15 22 13" src="https://github.com/elastic/kibana/assets/4283304/cb9466ca-d5a3-4c4d-a244-e019d014d051">
Lens:
<img width="306" alt="Screenshot 2024-03-14 at 15 22 50" src="https://github.com/elastic/kibana/assets/4283304/22563f58-f329-4d3d-b2e6-e3178cbbad13">

Discover:
<img width="305" alt="Screenshot 2024-03-14 at 15 23 18" src="https://github.com/elastic/kibana/assets/4283304/a52c11ef-ab6d-422d-ad3c-ceaaa03ec4ca">

Discover mobile:
<img width="407" alt="Screenshot 2024-03-14 at 15 23 42" src="https://github.com/elastic/kibana/assets/4283304/0d010731-4d7f-4a68-9c3a-582015d7ceef">

Reply:
I also made a small padding change for Lens, Michael gave me an ok (all the fields now have the same padding)

![Mar-14-2024 15-49-47](https://github.com/elastic/kibana/assets/4283304/b74b7a1b-fe0f-4f57-b33c-910390a00acb)